Client/Client.cc

```diff
@@ -158,6 +172,17 @@ TimeoutException::TimeoutException(const std::string& error)
 {
 }
 
+ConfigurationExceptionBad::ConfigurationExceptionBad(const std::string& error)
+    : Exception(error)
+{
+}
+
+ConfigurationExceptionChanged
+::ConfigurationExceptionChanged(const std::string& error)
```

Comment:
Yeah, I'm guilty of inconsistency in indents after `(\n`. Meh. The important thing to me here was that I was surprised to see a line beginning with `::`, as it's a little hard to disambiguate from `::` being used to select the topmost namespace (as in `::write(...)` to access libc's `write` function).